modules/nw-ovn-kubernetes-ansible-migration-about.adoc

```diff
@@ -0,0 +1,145 @@
+* You installed Docker.
+* You logged in to the {hybrid-console} and opened the link:https://console.redhat.com/ansible/ansible-dashboard[Ansible Automation Platfor] web console.
+* You have access to the `registry.ci.openshift.org` container image registry that is used by the {product-title} Continuous Integration (CI) functionality.
+
+.Procedure
+
+. Download the `network.offline_migration_sdn_to_ovnk` Ansible collection from the link:https://console.redhat.com/ansible/automation-hub[Ansible automation hub] web console page.
+
+. Install the `ansible-core` package, minimum version 2.15 and the `ansible-lint` package. The following example command shows how to install the `ansible-core` package on {op-system-base-full}:
++
+[source,terminal]
+----
+$ sudo dnf install -y ansible-core
+----
+
+* Install the `network.offline_migration_sdn_to_ovnk` Ansible collection by entering the following `ansible-galaxy` command:
++
+[source,terminal]
+----
+$ ansible-galaxy collection install network.offline_migration_sdn_to_ovnk
+----
+
+* Verify that the `network.offline_migration_sdn_to_ovnk` Ansible collection is installed on your system:
++
+[source,terminal]
+----
+$ ansible-galaxy collection list | grep network.offline_migration_sdn_to_ovnk 
+----
++
+.Example output
++
+[source,terminal]
+----
+# /home/<user>/.ansible/collections/ansible_collections <1>
+Collection                              Version
+------------------------                -------
+network.offline_migration_sdn_to_ovnk   2.0.0
+----
+<1> Where `<user>` is the user home directory on your system.
+
+. Disable auto-migration features by configuring the `playbooks/playbook-migration.yml` file:
++
+[source,terminal]
+----
+# ...
+    migration_disable_auto_migration: true
+    migration_egress_ip: false
+    migration_egress_firewall: false
+    migration_multicast: false
+    migration_mtu: 1400
+    migration_geneve_port: 6081
+    migration_ipv4_subnet: "100.64.0.0/16"
+# ...
+----
+where:
++
+`<migration_disable_auto_migration>`:: Disables the auto-migration of OpenShift SDN CNI plug-in features to the OVN-Kubernetes plugin. If you disable auto-migration of features, you must also set the `migration_egress_ip`, `migration_egress_firewall`, and ``migration_multicast` parameters to `false`. If you need to enable auto-migration of features, set the parameter to `false`. 
+`<migration_mtu>`:: Optional parameter that sets a specific maximum transmission unit (MTU) to your cluster network after the migration process.
+`<migration_geneve_port>`:: Optional parameter that sets a Geneve port for OVN-Kubernetes. The default port is `6081`.
+`<migration_ipv4_subnet>`:: Optional parameter that sets an IPv4 address range for internal use by OVN-Kubernetes. The default value for the OVN-Kubernetes plug-in is `100.64.0.0/16`.
+
+. After configuring the `playbook-migration.yml` file, enter the following `make` command to test the file against a `lint` rule:
```

Comment:
In your 

In the `network.offline_migration_sdn_to_ovnk` Automation Hub page explanation, you mention:

> For testing ansible and sanity tests on CI please use scripts test_incluster_lint.sh and test_incluster_sanity_lint.sh respectively

Do I need to document these scripts that can be used in place of the `make lint/sanity` command?

Reply:
no